testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1563,3 +1563,16 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+When(/^I copy autoinstall mocked files on server$/) do
+  target_dirs = "/autoinstall/Fedora_12_i386/images/pxeboot/,/autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/,/autoinstall/mock"
+  $server.run("mkdir -p {#{target_dirs}}")
```

Comment:
```suggestion
  target_dirs = ["/autoinstall/Fedora_12_i386/images/pxeboot/", "/autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/", "/autoinstall/mock"]
  $server.run("mkdir -p {#{target_dirs.join(' ')}}")
```

I have the feeling that you need to use a " " instead if "," to separate those dirs.
```
oscar@obarrios:/tmp> mkdir -p a/b,c/d,e/f
oscar@obarrios:/tmp> tree
.
├── a
│   └── b,c
│       └── d,e
│           └── f
```

Reply:
This solution fail my test. target_dirs in my PR is a string, not an array.

The final command line is mkdir -p {a/b,a/d,a/f} and was tested.

Reply:
Ah! I missed the `{}` I did not know that will behave like that :) Nice then